serde_arrow/src/internal/deserialization/deserializer.rs

```diff
@@ -0,0 +1,168 @@
+use serde::de::Visitor;
+
+use crate::internal::{
+    deserialization::outer_sequence_deserializer::OuterSequenceDeserializer,
+    error::{fail, Error, Result},
+};
+
+/// A structure to deserialize Arrow arrays into Rust objects
+///
+/// It can be constructed via
+///
```

Comment:
Incomplete comment?

Reply:
Only, if no features are active :). Once you have arrow or arrow2 configured the bullet list below is included, but I will also make this part of the comment conditional.